inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/pulsar/PulsarClientService.java

```diff
@@ -283,9 +284,9 @@ private void sendResponse(OrderEvent orderEvent, String inlongGroupId, String in
      */
     private void createConnection(CreatePulsarClientCallBack callBack) throws FlumeException {
         if (!pulsarClients.isEmpty()) {
+            logger.info("pulsar clients may already be initialized by config updating thread");
```

Comment:
this line may be needless, and I think we should remove it to avoid too logs if `pulsarClients` is not empty.